### PR TITLE
[CPO] Use cloud-provider-openstack-test-occm as base job

### PR DESCRIPTION
**Pull request by lingxiankong**
This change only affects cloud-provider-openstack-acceptance-test-e2e-conformance-stable-branch-v1.21

### CODE REVIEW

File: playbooks/cloud-provider-openstack-test-occm/run-1.19.yaml [hunk 1]
```diff
@@ -48,7 +48,7 @@
         name: install-k3s-vm
       vars:
         devstack_workdir: "{{ global_env.devstack_workdir }}"
-        cpo_branch: "{{ zuul.branch }}"
+        k8s_branch: "{{ k8s_release | default(zuul.branch) }}"
```
Comment (ramineni) [inline playbooks/cloud-provider-openstack-test-occm/run-1.19.yaml:51]: @lingxiankong etcd versions /go lang versions supported change for each k8s release. Does this prerequisites taken care by k3s?

Comment (lingxiankong) [reply playbooks/cloud-provider-openstack-test-occm/run-1.19.yaml:51]: What's the requirement for etcd and golang version? k3s doesn't use etcd. Why do we care about Golang? We have openstack-cloud-controller-manager built with minimum version of Go 1.15, and openstack-cloud-controller-manager talks with k8s API.

Comment (ramineni) [reply playbooks/cloud-provider-openstack-test-occm/run-1.19.yaml:51]: Ah, right. We are not building k8s unlike before, so shouldnt be required.

**Comment by ramineni** [comment]
looks good to me

**Comment by lingxiankong** [comment]
> looks good to me

Thanks for review, I will merge and restart zuul.

**Pull request merged**